cmd/crc/cmd/config/view.go

```diff
@@ -31,9 +31,9 @@ func init() {
 
 var configViewCmd = &cobra.Command{
 	Use:   "view",
-	Short: "Display complete crc configurations.",
-	Long: `Retrieves full list of crc configurations. Some of the configuration properties are equivalent
-to the options that you set when you run the 'crc start' command.`,
+	Short: "Display all set crc configuration properties",
+	Long: `Retrieves a full list of crc configuration properties and their values. Some of the configuration properties are equivalent
```

Comment:
"Full list" is actually incorrect here, only config props with assigned values are displayed.

Reply:
Also, retrieve sounds more like a database action than displaying them.

`Displays a list of ...`

likely a full list is meant here to also display 'default' options ?

perhaps 'assigned' is the least worst option here ;-).

Reply:
TBH, the second part of this sentence sounds weird. It does not really add information. If they are equivelant, what is the order in which they are interpreted? I know, but this is not really clear ... perhaps it is best to leave this out. @robin-owen WDYT?

Reply:
I feel the same way and think we just need to lose "Some of the configuration properties are equivalent
to the options that you set when you run the 'crc start' command." from the CLI.It's an interesting observation, but it belongs in the docs. 
And then why only some and not all? And as @gbraad mentioned, what order are interpreted in? But all that is a different rabbit hole to what we're doing here.

Reply:
I agree as well. I don't think the second sentence adds anything particularly valuable.

We can also mention the precedence of these options in the documentation.

Reply:
Documentation it is. It should actually be that CLI overrides config options.